cms/auth/tests/test_views.py

```diff
@@ -26,9 +27,17 @@ def _prep_request(self, req):
         SessionMiddleware(lambda r: None).process_request(req)
         req.session.save()
 
-        # attach a real FallbackStorage
-        storage = FallbackStorage(req)
-        req._messages = storage  # pylint: disable=protected-access
+        # Jake comment 2
+        # Attach a message storage backend to the request (typically FallbackStorage)
+        backend_path = settings.MESSAGE_STORAGE  # e.g. "django.contrib.messages.storage.fallback.FallbackStorage"
+        module_path, class_name = backend_path.rsplit(".", 1)
+
+        backend_module = import_module(module_path)
+        backend_cls = getattr(backend_module, class_name)
+
+        storage = backend_cls(req)
+        req._messages = storage  # pylint: disable=protected-access # attach to the request for messages.add_message()
```

Comment:
```suggestion
        backend_cls = import_string(settings.MESSAGE_STORAGE)
        req._messages = backend_cls(req)  # pylint: disable=protected-access # attach to the request for messages.add_message()
```

https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.module_loading.import_string